services/provider/server/server.go

```diff
@@ -155,6 +156,15 @@ func (s *OCSProviderServer) Start(port int, opts []grpc.ServerOption) {
 		klog.Fatalf("failed to listen: %v", err)
 	}
 
+	certFile := "/mnt/cert/tls.crt"
+	keyFile := "/mnt/cert/tls.key"
```

Comment:
This can get out of sync with the mount point added to the service spec. 
Can we have the mount point const in a shared file (maybe util file) and here we will just concatenate the file names?

Reply:
done